packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -214,6 +214,11 @@ export class AppStudioPluginImpl {
   }
 
   public async provision(ctx: PluginContext): Promise<Result<string, FxError>> {
+    const provisionProgress = ctx.ui?.createProgressBar(
```

Comment:
The first step is 0 which might be confusing to users, what about showing 1/1 for the progress bar?